app.py

```
@@ -1116,6 +1116,22 @@
+async def rag_stats():
+    """Get RAG system statistics."""
+    try:
+        from core.rag.simple_rag import get_rag_stats
+
+        stats = get_rag_stats()
+        return {
+            "enabled": str(os.getenv("FEATURE_RAG", "")).strip().lower()
+            in {"1", "true", "on", "yes"},
+            "stats": stats,
+        }
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)